scripts/prepareForFirstWeekLaunch.ts

```diff
@@ -1,30 +1,33 @@
+import chai from "chai"
+import { ethers } from "hardhat"
+import { solidity } from "ethereum-waffle"
+const { expect } = chai
+import {
+  GaugeController,
+  SDL,
+  GaugeHelperContract,
+  VotingEscrow,
+  Minter,
+  LiquidityGaugeV5,
+  GenericERC20
+} from "../build/typechain"
 import {
   BIG_NUMBER_1E18,
+  getCurrentBlockTimestamp,
   MAX_UINT256,
   asyncForEach,
-  getCurrentBlockTimestamp,
-  increaseTimestamp,
 } from "../test/testUtils"
-import {
-  GaugeController,
-  GaugeHelperContract,
-  GenericERC20,
-  Minter,
-  SDL,
-  VotingEscrow,
-} from "../build/typechain/"
-
-import chai from "chai"
-import { ethers } from "hardhat"
-import { solidity } from "ethereum-waffle"
-
-const { expect } = chai
-
 chai.use(solidity)
 
+/*  tldr:
+  *  sets gauges weights, deploys and mints dummy reward tokens, adds dummy tokens
+  *  as rewards to gauges, gets LPtoken by depositing to USDv2 pool, transfers 
+  *  lp tokens to signers[1] and signers[2], transfer SDL to signers[1], signers[1]
```

Comment:
Should this be transfer SDL to signers[1] and signers[2]?

Also I believe our engineers only have private key for the first hardhat address but not the subsequent ones. Will need to share that out for testing purposes.